src/codeflare_sdk/templates/base-template.yaml

```diff
@@ -16,21 +16,21 @@ spec:
       custompodresources:
       - replicas: 1
         requests:
-          cpu: 2
```

Comment:
Are the template changes expected?

Reply:
Thanks for pointing that out. I can't remember why I had to modify those values for the test, I think it was until another PR needed to get merged. I reverted the values and the test continues to pass now.